x/ecocredit/basket/date_criteria_test.go

```diff
@@ -29,3 +30,82 @@ func TestDateCriteriaToApi(t *testing.T) {
 	require.NotNil(dw)
 	require.Equal(durStd, dw.AsDuration(), "handles window date")
 }
+
+func TestValidateDateCriteria(t *testing.T) {
```

Comment:
Moved from `msg_create_test.go` to `date_criteria_test.go` without changes the test cases.